Include section about large skyboxes

stefanpl opened the pull request:
If I understand them correctly, I think both options (maxZ and infiniteDistance) have their uses and should be mentioned here.

I've been working with a skyboxed scene for some time, and finally took the time to investigate why it didn't quite behave as I wanted. [This old SO post](https://stackoverflow.com/questions/35387282/how-to-make-a-very-large-skybox-babylon-js) was all the info I needed.

deltakosh commented on content/extensions/Viewer/Configuring_the_viewer.md:
```diff
@@ -282,177 +282,35 @@ And finally, we need to tell the manager which mapper to use:
-        onSceneInit?: string;
-        onModelLoaded?: string;
-    }
+This will word as the default mapper is the str-to-json mapper. If you define your own type, you could set the mapper (important - after registering it):
```
Work

deltakosh replied:
Actually I had hard times understanding the sentence :)

RaananW replied:
it's meant to be cryptic!

ok .... I will rephrase that :)

stefanpl committed:
Include section about large skyboxes

I think both options (maxZ and infiniteDistance) have their uses and should be mentioned here.

Been working with a skyboxed scene for some time, and finally took the time to investigate why it didn't quite behave as I wanted. Found the answers [here](https://stackoverflow.com/questions/35387282/how-to-make-a-very-large-skybox-babylon-js).

RaananW reviewed (commented):
Whereas I understand the need for that, it should just be a comment in the code that creates the skybox. It has less to do with the infiniteDistance variable and more to do with the user's scene configuration. A skybox that is larger than 1000 is probably needed, if your scene size is larger than 1000 units.

You can add a sentence saying "when creating the skybox make sure that your camera's maxZ is adjusted to the size of the skybox"

sebavan commented:
@stefanpl can you add the requested change ? I will merge right after.

sebavan commented:
Closing due to inactivity, let s reopen if updated ?
